### DISCUSSION TITLE
Add support for transferring issues across repositories on the mobile app

### DISCUSSION THREAD

**Discussion by TanayParikh** [community/community#10295] [Mobile]
It's possible to transfer issues in the web UI, it would be nice to be able to do this via the app as well as part of the issue management process.

**Comment by brianlovin** [comment]
Thanks for the note @TanayParikh — agreed! We have an open tracking issue for this, and I'll add your +1 so that we can try to prioritize this in the future.

**Reply by SableRaf** [reply]
Any updates on this? Transferring issues on mobile would be very helpful for maintainers on the go. Having this in the three dot menu would make a lot of sense. Curious if this is still on the roadmap or if there’s any recent discussion about it.
